registry-config.yaml

```diff
@@ -0,0 +1,41 @@
+# https://docs.docker.com/registry/configuration/
+
+version: 0.1
+log:
+  level: debug
+  fields:
+    service: registry
+storage:
+  cache:
+    blobdescriptor: inmemory
+  filesystem:
+    rootdirectory: /mnt/docker0storage/registry
+http:
+  addr: 443:5000 # https://docs.docker.com/registry/configuration/#letsencrypt
+  host: docker.cdot.systems
```

Comment:
Could we change it to ${DOMAIN} like what @manekenpix suggested in one of the other file?

Reply:
If we do, we'll have to use `envsubst` or something to replace the values before we use this file.